.buildkite/src/Command/Dsl.dhall

```diff
@@ -58,7 +58,7 @@ let targetToAgent = \(target : Size) ->
 let build : Config.Type -> Command/Base.Type = \(c : Config.Type) ->
   Command/Base.build Command/Base.Config::{
     agents =
-      let agents = targetToAgent c.target in
+      let agents = targetToAgent c.target # toMap { local = "local" } in
```

Comment:
Don't approve this PR until I back this change out -- this is to force the jobs to run on my laptop.